lib/ui/window.dart

```diff
@@ -407,7 +407,7 @@ class SingletonFlutterWindow extends FlutterView {
     'This feature was deprecated after v3.7.0-32.0.pre.'
   )
   SingletonFlutterWindow._() : super._(
-    _kImplicitViewId,
+    _implicitViewId!,
```

Comment:
Will this result in an error if the implicit view is disabled, even if the user never uses the `window` global? We should be able to disable the implicit view without causing errors, even in phase 1. What are our other options?

Reply:
I had the same question, but I have the unit test `ShellWithImplicitViewDisabledStartsWithoutImplicitView` that disables the implicit view, and it passes.

Originally I had the global `window` variable marked as `late`, but Dart marked the `late` unnecessary. I guess global object variables are late inherently.

Reply:
We discussed last Thursday and this is our thought process and conclusion (to my understanding).
* The engine will be allowed to disable implicit views at the end of phase 2, which is earlier than the official removal of `window`.
* Using `window` in whatever ways should not trigger errors when the implicit view is disabled.
* Instead of mapping `window` to an arbitrary existing view, which should provide expected behaviors: the `window` corresponds to the implicit view, such as `FlutterEngine.viewController`.
* Therefore, using `window` when the implicit view is disabled/closed should be no-ops.